cli/admin/src/test/java/io/pravega/cli/admin/controller/AuthEnabledControllerCommandsTest.java

```diff
@@ -0,0 +1,252 @@
+                new RetryOneTime(5000));
+        curatorFramework.start();
+        Assert.assertNotNull(command.instantiateSegmentHelper(curatorFramework));
+
+        // Try the Zookeeper backend, which is expected to fail and be handled by the command.
+        Properties properties = new Properties();
+        properties.setProperty("cli.store.metadata.backend", CLIControllerConfig.MetadataBackends.ZOOKEEPER.name());
+        STATE.get().getConfigBuilder().include(properties);
+        commandArgs = new CommandArgs(Arrays.asList(scope, testStream), STATE.get());
+        new ControllerDescribeStreamCommand(commandArgs).execute();
+        properties.setProperty("cli.store.metadata.backend", CLIControllerConfig.MetadataBackends.SEGMENTSTORE.name());
+        STATE.get().getConfigBuilder().include(properties);
+    }
+
+    @Test
+    public void testAuthConfig() throws Exception {
+        SETUP_UTILS.createTestStream("testListScopesCommand", 2);
+        Properties pravegaProperties = new Properties();
+        pravegaProperties.setProperty("cli.security.auth.enable", "true");
+        pravegaProperties.setProperty("cli.security.auth.credentials.username", "admin");
+        pravegaProperties.setProperty("cli.security.auth.credentials.password", "1111_aaaa");
+        STATE.get().getConfigBuilder().include(pravegaProperties);
+        String commandResult = TestUtils.executeCommand("controller list-scopes", STATE.get());
+        // Check that both the new scope and the system one exist.
+        Assert.assertTrue(commandResult.contains("_system"));
+        Assert.assertTrue(commandResult.contains(SETUP_UTILS.getScope()));
+        Assert.assertNotNull(ControllerListScopesCommand.descriptor());
+        // Restore config
+        pravegaProperties.setProperty("cli.security.auth.enable", "false");
+        STATE.get().getConfigBuilder().include(pravegaProperties);
+
+        // Exercise response codes for REST requests.
+        CommandArgs commandArgs = new CommandArgs(Collections.emptyList(), new AdminCommandState());
+        ControllerListScopesCommand command = new ControllerListScopesCommand(commandArgs);
+        command.printResponseInfo(Response.status(Response.Status.UNAUTHORIZED).build());
+        command.printResponseInfo(Response.status(Response.Status.INTERNAL_SERVER_ERROR).build());
+    }
+
+    static String executeCommand(String inputCommand, AdminCommandState state) throws Exception {
+        Parser.Command pc = Parser.parse(inputCommand);
+        Assert.assertNotNull(pc.toString());
+        CommandArgs args = new CommandArgs(pc.getArgs(), state);
+        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        TestingDescribeStreamCommand cmd = new TestingDescribeStreamCommand(args);
+        try (PrintStream ps = new PrintStream(baos, true, "UTF-8")) {
+            cmd.setOut(ps);
+            cmd.execute();
+        }
+        return new String(baos.toByteArray(), StandardCharsets.UTF_8);
```

Comment:
Please be consistent in how you reference UTF8. Here you use `StandardCharset`, yet 4 lines above you use a string.

Reply:
The references are consistent now.